updating/preparing_for_updates/updating-cluster-prepare-past-4.18.adoc

```diff
@@ -0,0 +1,128 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="updating-cluster-prepare-past-4.18"]
+= Preparing to update from {product-title} 4.18 to a newer version
+include::_attributes/common-attributes.adoc[]
+:context: updating-cluster-prepare-past-4.18
+
+toc::[]
+
+Learn about some of the specific concerns around {op-system-base} compute machines before attempting to update from {product-title} 4.18 to a newer version.
+
+[id="migrating-workloads-to-different-nodes_{context}"]
+== Migrating workloads to different nodes
```

Comment:
Can we include something like "package-based {op-system-base} worker nodes" in this header somehow to make the scoping more obvious?  I'm hoping for some way for folks who know they only have RHCOS nodes to know they can skip this section.

Reply:
totally